README.md

```diff
@@ -1574,6 +1876,44 @@ $posts = Post::whereHas('user', function ($query) use ($request) {
 
 Tip given by [@adityaricki](https://twitter.com/adityaricki2)
 
+### Update an existing pivot record
+If you want to update an existing pivot record on the table, use `updateExistingPivot` instead of `syncWithPivotValues`.
+
+```php
+// Migrations
+Schema::create('role_user', function ($table) {
+    $table->unsignedId('user_id');
+    $table->unsignedId('role_id');
+    $table->timestamp('assigned_at');
+})
+
+// first param for the record id
+// second param for the pivot records
+$user->roles()->updateExistingPivot(
+    $id, ['assigned_at' => now()],    
+);
+```
+
+Tip given by [@sky_0xs](https://twitter.com/sky_0xs/status/1461414850341621760)
+
+### Relation that will get the newest (or oldest) item
```

Comment:
@krekas need to add a sentence that it's available from some new Laravel version, was it 8.49 or something? Please google.